src/util/panic_hook.rs

```diff
@@ -63,7 +64,7 @@ fn track_hook(p: &PanicInfo) {
 }
 
 /// Exit the whole process when panic.
-pub fn set_exit_hook(panic_abort: bool) {
+pub fn set_exit_hook(panic_abort: bool, guard: Option<GlobalLoggerGuard>) {
```

Comment:
I prefer to define an OnHook trait for more customization, not only for Logger here.